src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json

```diff
@@ -28,6 +28,5 @@
   },
   "Slack": {
     "WebHookUrl": ""
-  },
-  "UseTableStorageQueryStore": true
```

Comment:
We still need `UseTableStorageQueryStore` in this file as it is different to what is in appSettings.json

Reply:
restored it.